content/docs/join_us/recruitment_FAQs.md

```diff
@@ -90,29 +141,82 @@ Also, it is important to highlight:
+The team assists the Turing community with designing, building, deploying, and maintaining research infrastructures, as well as providing service delivery and user support for the Institute’s research computing platforms.
+This is an opportunity for a technically-skilled professional with a good understanding of either cloud or high-performance computing (HPC), experience with both application and infrastructure, and an interest in facilitating and enhancing world-leading data science and AI research.
+You will build tools and automation to eliminate mundane operations and craft repeatable processes.
+As part of the team, you will also work with other research software engineers and data scientists on various projects and will participate in all phases of cloud / HPC development, from requirements-capture, definition, and design; to development, deployment, and maintenance; to performance tuning and monitoring.
+You are also expected to keep up-to-date with the latest cutting-edge developments, and/or make use of the UK’s most advanced computing facilities like Dawn, ARCHER 2, JADE 2, and Baskerville.
 
 ### What is the difference between a Junior and a Standard role?
 
 The main difference between the Junior and Standard roles is that the Junior is a training position for becoming (in 1-2 years) a Standard research data scientist or software engineer.
 We consider candidates for the Standard position if they have a PhD or an equivalent working experience (meaning for instance a few years of work in a data scientist or software engineer position, in close contact with research projects).
 If you think you have enough experience you can apply to the Standard position and stress this in your cover letter.
 You can also apply to the Junior position and mention in your cover letter that you are also considering the Standard position, in case you feel your profile or working experience is in between the two.
-Feel free to get in touch with the REG recruitment team ([reg-recruitment-inbox@turing.ac.uk](mailto:reg-recruitment-inbox@turing.ac.uk)) about this point.
+Feel free to get in touch with the REG recruitment team ([reg-recruitment-inbox@turing.ac.uk][recruitment]) about this point.
+
+### What kind of projects do you work on in REG?
+
+The projects at the Turing largely fall into three themes, namely Environment and Sustainability; Health; and Defence and National Security.
+These projects tend to revolve around the _application_ of data science and AI to solve challenges in these areas.
+Apart from this, there are also projects in foundational research in data science and AI.
+
+You can see a list of projects we are currently working on on [our website](https://alan-turing-institute.github.io/REG/).
+
+### Are working hours flexible?
+
+Formally, the Turing's core working hours are between 10 and 4.
+Most people would work these hours plus some additional time on either side.
```

Comment:
I would add something like "to make up their 36 hour working week.", as I think that is what is in our contract